atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/SimpleDbWriteTable.java

```diff
@@ -67,8 +71,14 @@ public void put(Collection<Map.Entry<Cell, Value>> data) {
 
     private void put(List<Object[]> args) {
         try {
-            conns.get().insertManyUnregisteredQuery("/* INSERT_ONE (" + tableName + ") */"
-                    + " INSERT INTO " + prefixedTableName() + " (row_name, col_name, ts, val) "
+            //        if (config.type().equals(OracleDdlConfig.TYPE)) {
+            //            return ((OracleDdlConfig) config)
+            //                    .tableNameMapper()
+            //                    .getShortPrefixedTableName(config.tablePrefix(), tableRef);
+            //        }
+            //        return config.tablePrefix() + tableRef;
+            conns.get().insertManyUnregisteredQuery("/* INSERT_ONE (" + tableRef + ") */"
+                    + " INSERT INTO " + getInternalTableName() + " (row_name, col_name, ts, val) "
```

Comment:
Looks like `getInternalTableName()` does a select to get table name mapping, which will kill throughput. We should be able to cache the mappings.